### PR TITLE
express-post-json

### PR DESCRIPTION
**Pull request by CristoferMar**
express-post-json

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+app.post('/api/grades', (req, res) => {
+  req.body.id = nextId;
+  nextId++;
+  grades = req.body;
```
Comment (thebearingedge) [inline express-post-json/index.js:15]: The new grade object should be stored at its id in the grades object. Posting multiple grades should retain them all until the server is restarted.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: express-post-json/index.js (hunk 1 above)
Comment (CristoferMar) [reply express-post-json/index.js]: express-post-json

File: express-post-json/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+const grades = {};
+let nextId = 1;
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
```
Comment (thebearingedge) [inline express-post-json/index.js:10]: Ok, now this needs to get updated so that it responds with an array, like in `express-get-json`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by CristoferMar** [comment]
image

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**